### PR TITLE
feat(mediapro / downloadpro): add amazon info

### PR DESCRIPTION
**Pull request by ameli90**
https://github.com/JOOlanders/documentation/issues/74

### CODE REVIEW

**Comment by miljan-aleksic** [comment]
Move the element docs to **Advanced/Amazon S3** section.

**Comment by ameli90** [comment]
done. I have made a partial but cannot test on localhost it works properly. Can we check it on live?

File: 01.elements/mediapro/02.advanced/03.amazon-s3/docs.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Upgrading to Pro
```
Comment (miljan-aleksic) [inline 01.elements/mediapro/02.advanced/03.amazon-s3/docs.md]: Fix title

Comment (ameli90) [inline 01.elements/mediapro/02.advanced/03.amazon-s3/docs.md]: fixed

**Comment by miljan-aleksic** [comment]
Will merge and test the partial asap. Tks.

**Pull request merged**